src/core/agent/services/multiSigService.ts

```diff
@@ -716,6 +722,149 @@ class MultiSigService extends AgentService {
     }
     return true;
   }
+
+  async getMultisigParticipants(multisigSignifyName: string) {
+    const members = await this.props.signifyClient
+      .identifiers()
+      .members(multisigSignifyName);
+    const multisigMembers = members["signing"];
+    const allMultisigMemberIdentifiers =
+      await this.identifierStorage.getMultisigMemebersMetadata();
+    const owningMember = multisigMembers.find((member: any) =>
+      allMultisigMemberIdentifiers.some(
+        (identifier) => member.aid === identifier.id
+      )
+    );
+
+    if (!owningMember) {
+      throw new Error(MultiSigService.MEMBER_AID_NOT_FOUND);
+    }
+    const ourIdentifier = allMultisigMemberIdentifiers.find(
```

Comment:
Since we iterated above to find `owningMember`, can we not just find `ourIdentifier` in the same loop?